cmd/jujud/updateseries/updateseries.go

```diff
@@ -92,6 +85,12 @@ func (c *UpdateSeriesCommand) Init(args []string) error {
 		return errors.Errorf("cannot run on a controller machine")
 	}
 
+	if c.manager == nil {
+		c.manager = service.NewSystemdServiceManager(systemd.IsRunning)
+		logger.Criticalf("Initialized the manager......")
```

Comment:
We don't want these log messages in the code that lands. (That's one of the reasons I log messages as critical when I'm trying to debug something - it's easy to find them to remove later before I commit.)